templates/webapps/galaxy/workflow/run_complete.mako

```diff
@@ -0,0 +1,55 @@
+<%inherit file="/base.mako"/>
```

Comment:
I merged latest dev into this branch and it is still here.
We modified this file at a time were `run_complete.mako` still existed but it should not be in this branch anymore :)
Any help appreciated! I fear a rebase as so many JS changes are in that a rebase is painful.